pkg/sql/compile/compile.go

```diff
@@ -472,6 +471,25 @@ func (c *Compile) compilePlanScope(n *plan.Node, ns []*plan.Node) ([]*Scope, err
 			return nil, err
 		}
 		return ss, nil
+	case plan.Node_UNNEST:
+		var (
+			pre []*Scope
+			err error
+		)
+		if len(n.Children) != 0 {
```

Comment:
No matter what the case, there should be children, just the difference between value scan or table scan.

Reply:
🫡